src/atom/valence/mod.rs

```diff
@@ -5,7 +5,7 @@ use crate::atom::core::prelude::*;
 /// This function is called by [`crate::main`].
 pub(crate) fn main() -> Result<()> {
     let settings = Settings::get();
-    let config = Config::get(&settings);
+    let config = Config::try_get(settings.config_path()).expect("Failed to parse config file");
```

Comment:
`.expect()` panics and gives pretty ugly errors, that error message should be somewhere in `core` and just a `?` in valence